lib/src/main/java/me/onebone/toolbar/FabPosition.kt

```diff
@@ -0,0 +1,19 @@
+package me.onebone.toolbar
+
+
+@Suppress("INLINE_CLASS_DEPRECATED", "EXPERIMENTAL_FEATURE_WARNING")
+inline class FabPosition internal constructor(@Suppress("unused") private val value: Int) {
```

Comment:
What is the `value` property for? Is it inappropriate to replace it with enum?

Reply:
That was in official material's scaffold component. I copied this class from there.
I can change it to Enum if you want.

Reply:
I'm purely curious why they made that decision. Possibly reducing the size of dex?
For now, though, I would like to change it to enum for the favor of enforcing exhaustive branches 😃

Reply:
Changed to Enum in latest commit